src/webapi/rendering_context.rs

```diff
@@ -15,6 +17,20 @@ pub trait RenderingContext {
 /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D)
 pub struct CanvasRenderingContext2d(Reference);
 
+#[derive(Debug)]
```

Comment:
For enums like these we also would want `Copy`, `Clone`, `PartialEq`, `Eq` derived.

Reply:
Also missing docs. (: